new-site/src/components/LiveErrorCore.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import { StaticConfigLoader, HtmlValidate } from "html-validate/dist/cjs/browser";
+import { default as html5 } from "html-validate/elements/html5.json";
+
+import './LiveErrorCore.scss'
+
+const loader = new StaticConfigLoader({
+  extends: ["html-validate:standard"],
+  elements: [html5],
+});
+const htmlvalidate = new HtmlValidate(loader);
+
+const LiveErrorCore = ({code}) => {
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const report = htmlvalidate.validateString(code);
+    if (!report.valid) {
```

Comment:
It might be a matter of taste, but I like it more when boolean value is used in condition as it is:
```js
if (report.valid) {
  setError(null);
} else {
  setError(report.results[0].messages[0]);
}
```

Reply:
- [ ] Change boolean value as it is in condition